content/posts/fluent-bit-surpasses-10-billion-docker-pulls.md

```diff
@@ -1,92 +1,42 @@
 ---
 title: "Fluent Bit surpasses 10 billion Docker pulls!"
-date: "2023-09-11"
+date: "2023-09-25"
 description: "That’s billion, with a “B.” It is currently averaging over 23 million pulls per day. That’s 270 pulls every second, 24/7"
```

Comment:
I mean if we're being pedantic it is a US billion, i.e. a thousand million, rather than a million million...

I will concede it has generally moved to the short scale vs long scale though :) 
https://en.wikipedia.org/wiki/Billion